### ISSUE TITLE
v0.0.0.65 SN status uncertain

### ISSUE THREAD

**Issue by walkjivefly** [Crowndev/crown#174]
A systemnode for which a `startsystemnode` was issued appears in the systemnode list in ENABLED status but does not show the expected "HotCold" node activated message in debug.log and reports garbled status in `getsystemnodestatus`.

activation attempted from collateral wallet:
```
mark testnet3 $ tcrowncli startsystemnode missing false
{
  "overall": "Successfully started 2 systemnodes, failed to start 0, total 2",
  "detail": [
    {
      "alias": "sn01",
      "result": "success",
      "error": ""
    },
    {
      "alias": "sn02",
      "result": "success",
      "error": ""
    }
  ]
}
```

The node shows as ENABLED in the SN list
```
crown@crown-testnet-04:~/.crown/testnet3$ crown-cli listsystemnodes
[
  {
    "rank": 1,
    "txhash": "264c53c012813ce9d8d2aeea54d76e426951dcf16affabf0fdc15db37fb1d9d7",
    "outidx": 1,
    "pubkey": "048c5cf78d01295b9ada8c55ded24b7f40058564e7f28e1d87cec76808de5b73dd3783cebf5a494ab0eec86a12f7cd1eb4c91b41868297a4d0fd6093cd323a0203",
    "status": "ENABLED",
    "addr": "tCRWS8UZUvnYjViSBodpdt4uWt2auS1YB2xnB",
    "version": 70058,
    "ipaddr": "92.60.46.27:18333",
    "lastseen": 1667252930,
    "activetime": 0,
    "lastpaid": 0
  },
  {
    "rank": 2,
    "txhash": "31ba10df1c32509eaf147e8d94c1b87ea590909b589326eda4ada94ba4082339",
    "outidx": 1,
    "pubkey": "0465de4f6a53282b8cbfe555f8dba1e6d9696ed72d48c7e4d4ab0c4996735a19e6ea765acdf2122be7343bafdba659a14e6f310627a5b492d6f54e3279fbe2ee48",
    "status": "ENABLED",
    "addr": "tCRWHj6yE88Txjm2QupUBHxZG8uXVKkVew1oe",
    "version": 70058,
    "ipaddr": "92.60.45.239:18333",
    "lastseen": 1667253060,
    "activetime": 0,
    "lastpaid": 0
  },
  {
    "rank": 3,
    "txhash": "75e466d6ca8be03f5162387e0dab4d19fbcab817321705dfe5cd2b2f47c11343",
    "outidx": 1,
    "pubkey": "04018f62587d2b362489547f7809d82e06d16009e709f401b630f36ca8a46aada90b6146e3837cde213f1174ad1c1d45b6e3f60568aab6615ea6bcd15dced8cd3e",
    "status": "ENABLED",
    "addr": "tCRWD28C34bpHEkJ3uH9bt2V1zEjSJURwXgNH",
    "version": 70058,
    "ipaddr": "92.60.45.238:18333",
    "lastseen": 1667253059,
    "activetime": 0,
    "lastpaid": 0
  }
]
```
but when asked to report status:
```
crown@crown-testnet-04:~/.crown/testnet3$ crown-cli getsystemnodestatus
error code: -1
error message:
Systemnode not found in the list of available systemnodes. Current status: Not capable systemnode: Invalid port: 18333 - 9340 is only supported on mainnet.
```

**Comment by walkjivefly** [comment]
After restarting the node, the status (at the node itself, where else would `getsystemnodestatus` run?) is
```
crown@crown-testnet-04:~/.crown/testnet3$ crown-cli getsystemnodestatus
error code: -1
error message:
Active Systemnode not initialized.
```

In the debug.log following the most recent start attempt is
```
2022-11-01T21:54:38Z CActiveSystemnode::ManageStatus() - not capable: Invalid port: 18333 - 9340 is only supported on mainnet.
```

Here's the full debug.log
[d.zip](https://github.com/Crowndev/crown/files/9914238/d.zip)

**Comment by walkjivefly** [comment]
This seems to have been resolved in 0.0.0.69
